### ISSUE TITLE
Turn off some shortcuts

### ISSUE THREAD

**Issue by Son-kun** [nylas/N1#513]
Now it's imposible to write mails in Nylas with character like "alt + l"(ł) and "alt + s"(ś) in polish language.
I can't find place to turn off or change them in Nylas.

**Comment by bengotow** [comment]
Hi @Son-kun — thanks for downloading N1! What happens when you press those characters? It should definitely be possible to edit your keymaps, but we shouldn't have anything bound to `alt` - just `cmd` or `ctrl`, depending on your platform.

**Comment by Son-kun** [comment]
That shortcuts is nowhere defined, but alt+l work as closing current window.
alt+s don't do anything what I can notice on GUI.

**Comment by rafpran** [comment]
Check https://github.com/nylas/N1/blob/37e3fe0f4533ee5139cf0c27ee8cf9a008bc4ffe/keymaps/base-win32.cson
Alternative keymaps in windows and i18n linux are connected to right alt or left alt + ctrl. It is not a good idea to make ANY shortcut based on these keys.
It should be connected with left alt only or shift+alt. 

ALL shortucts should be editable, and no shortcut should be active on setup screen (eg. cannot type name Rafał <last letter is L + right alt>).

**Comment by pawelkaczmarek** [comment]
+1 for this issue. I tried Nylas and thought it will finaly replace Gmail in browser. Unfortunately it's unusable in Polish language since I can't write diacritics using alt on keyboard.

**Comment by bengotow** [comment]
(Related http://www.polishnewsseattle.org/info/PolskieLitery.htm#4)

**Comment by Son-kun** [comment]
Ok, then we know where is problem, my question is what we do with this?
It's a way to define in that shortcuts with left alt?

**Comment by pawelkaczmarek** [comment]
These commands are the cause of this issue (ctrl-alt is detected as right alt):

```
  ### N1 developer commands. ###
  'cmdctrl-alt-l': 'window:reload'
  'cmdctrl-alt-s': 'application:run-all-specs'
```

I tried to overwrite them using keymap.cson. ``cmdctrl-alt-l`` now works fine, but rule for ``cmdctrl-alt-s`` is ignored. Similar problem exists in Atom and there is a [keyboard-localization](https://atom.io/packages/keyboard-localization) package that soves it.

**Comment by emorikawa** [comment]
This should be fixed in 14edd8a
We shouldn't be overriding the alt key at all really in Windows.
